src/app/services/app-updater.service.ts

```diff
@@ -0,0 +1,68 @@
+import { ApplicationRef, Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material';
+import { SwUpdate } from '@angular/service-worker';
+import { concat, interval } from 'rxjs';
+import { first } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AppUpdaterService {
+  isSnackbarloaded = false;
+  checkUpdateInterval = 1000 * 60 * 7;
```

Comment:
7 minutes is far too often. Let's do every 2 hours (which is still probably too much).